src/plugins/elasticsearch.js

```diff
@@ -21,9 +21,15 @@ function createWrapRequest (tracer, config) {
         }
       })
 
-      if (JSON.stringify(params.body)) {
-        span.setTag('elasticsearch.body', JSON.stringify(params.body))
+      const indexPath = JSON.stringify(params.path).split('/')[1]
+
+      if (indexPath.substring(0, 1) !== '_') {
+        span.setTag('elasticsearch.index', indexPath)
+      }
+      if (params.body && params.body.query) {
+        span.setTag('db.statement', JSON.stringify(params.body.query).substring(0, 1024))
```

Comment:
We are losing a lot of information by restricting this to the query field.  Can you use the whole body?